finance/funding_requests.md

```diff
@@ -0,0 +1,52 @@
+
+Funding requests can be made to the Astropy Finance Committee.  The following provides a description for how to obtain funding for different types of activities. 
+
+
+## Travel
+
+1. Send an email to astropy-finance-committee@googlegroups.com with the following information:
+* Name
+* Description
+* Total Request
+* Itemized Budget
+
+Costs for per diem can use the [US Government per deim rates | https://www.gsa.gov/travel/plan-book/per-diem-rates], and airfare costs should be justified. 
```

Comment:
```suggestion
Costs for per diem can use the [US Government per diem rates | https://www.gsa.gov/travel/plan-book/per-diem-rates], and airfare costs should be justified. 
```

Not clear to me from the above — do we expect people to claim the federal per diem, or do they have the option of submitting itemized receipts? In fact, are we just using the per diem rates to estimate expenses, and will still require receipts before reimbursement?

Reply:
Also — do we use the State Department rates for non-US travel? They are often _very_ high.